src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -6830,20 +6830,23 @@ namespace HVACUnitarySystem {
 						CalcUnitarySystemToLoad( UnitarySysNum, AirLoopNum, FirstHVACIteration, CoolPLR, HeatPLR, OnOffAirFlowRatio, TempSensOutput, TempLatOutput, HXUnitOn, _, _, CompressorONFlag );
 					} else if ( CoolingLoad ) {
 						// RegulaFalsi may not find cooling PLR when the latent degradation model is used.
-						// IF iteration limit is exceeded, find tighter boundary of solution and repeat RegulaFalsi
+						// IF iteration limit is exceeded (SolFlag = -1), find tighter boundary of solution and repeat RegulaFalsi
 						TempMaxPLR = -0.1;
 						TempSysOutput = SensOutputOff;
 						TempLoad = ZoneLoad;
-						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 1.0 ) {
+						while ( ( TempSysOutput - TempLoad ) > 0.0 && TempMaxPLR < 0.95 ) { // avoid PLR > 1 by limiting TempMaxPLR to 1 (i.e., TempMaxPLR += 0.1)
 							// find upper limit of HeatingPLR
 							TempMaxPLR += 0.1;
```

Comment:
OK.  I'll test it as-is.  Just seems more complicated that it needs to be. As for clock cycles, it's a wash, because the max on 1.0 would replace the if block below.